docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -250,8 +262,52 @@ The sub-aim for this project is to create a 'power calculation' of how many pati
 5.5 Output meta-table shell
 ---------------------------
 
-.. todo::
-  - add special stratifications if necessary
+.. list-table:: Output shell table
+   :header-rows: 1
+
+   * - Location
+     - Year
+     - Age group
+     - Sex
+     - Poulation group
+     - Scenario
```

Comment:
Is 'MM Stage' of interest to the client and should be added to the output shell table?

Reply:
Depending on what we promised, we can combine clinical MM and RRMM in reporting outcome of rates, or split by MM stage. @aflaxman What you suggest?

Reply:
Include stage